src/main/java/frc/core/util/DoubleButton.java

```diff
@@ -0,0 +1,18 @@
+package frc.core.util;
+
+import edu.wpi.first.wpilibj2.command.button.Button;

```

Comment:
https://github.com/wpilibsuite/allwpilib/blob/main/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/pacgoat/triggers/DoubleButton.java

Reply:
@gluvielmo faz q nem a first fez o exemplo, eles passam o joystick pra dentro do double button

Reply:
No final eles tem que passar os botões como int, o que dificulta o entendimento na minha opinião, neste link https://www.chiefdelphi.com/t/can-you-bind-a-command-to-only-activate-when-2-buttons-are-held/347368/4 mostram como fazer com o Button mesmo, não acha que podemos fazer assim? @gabrielatomaz

Reply:
eles passaram como int por preguiça, pq o enum é um int também (Button.kA.value, value retorna o int do button kA).
fazendo o double button passando o joystick mantem o padrão da FIRST

```
new JoystickButton( // instancia o joystick, no caso seria o DoubleButton
     this.operator, // passa o joystick, no caso, o operator
     Button.kBumperRight.value // passa o valor em int do controle por enum (aí só no caso, tu vai passar dois)
);
```